docs/03-developer-tools/00-sugar/03-guides/04-sugar-for-cmv3.md

```diff
@@ -0,0 +1,564 @@
+          :.. label: OGslic
+          :   :.. bot tax: none
+          :   :.. sol payment: 
+          :   :   :.. lamports: 1 (◎ 0.000000001)
+          :   :   :.. destination: PanbgtcTiZ2PveV96t2FHSffiLHXXjMuhvoabUUKKm8
+          :   :.. token payment: none
+          :   :.. start date: 
+          :   :   :.. date: Thu October 20 2022 12:00:00 UTC
+          :   :.. third party signer: none
+          :   :.. token gate: 
+          :   :   :.. amount: 1000000000
+          :   :   :.. mint: 7nE1GmnMmDKiycFkpHF7mKtxt356FQzVonZqBWsTWZNf
+          :   :.. gatekeeper: none
+          :   :.. end date: none
+          :   :.. allow list: none
+          :   :.. mint limit: none
+          :   :.. nft payment: none
+          :   :.. redeemed amount: none
+          :   :.. address gate: none
+          :   :.. nft gate: none
+          :   :.. nft burn: none
+          :   :.. token burn: none
+          :
+          :.. label: Public
+              :.. bot tax: none
+              :.. sol payment: 
+              :   :.. lamports: 2000000000 (◎ 2)
+              :   :.. destination: PanbgtcTiZ2PveV96t2FHSffiLHXXjMuhvoabUUKKm8
+              :.. token payment: none
+              :.. start date: 
+              :   :.. date: Thu October 20 2022 18:00:00 UTC
+              :.. third party signer: none
+              :.. token gate: none
+              :.. gatekeeper: none
+              :.. end date: none
+              :.. allow list: none
+              :.. mint limit: none
+              :.. nft payment: none
+              :.. redeemed amount: none
+              :.. address gate: none
+              :.. nft gate: none
+              :.. nft burn: none
+              :.. token burn: none
+
+✅ Command successful.
+```
+
+</p>
+</details>
+
+### `remove`
+
+To remove a Candy Guard from a Candy Machine, use the command:
+
+```
+sugar guard remove
+```
+
+<details>
+<summary>Output</summary>
+<p>
+
+```
+[1/1] 🔩 Unwrapping
+Signature: 5hbySp2JPuFrdjauDFVu5P4Fa3EkV9pAyqtgqDRs38d6aVwgiSpsBEs33BKQjqPMvdyyteS2W7ApfmcGstRuajNz
+
+The candy guard is no longer the mint authority of the candy machine.
+  -> New mint authority: PanbgtcTiZ2PveV96t2FHSffiLHXXjMuhvoabUUKKm8
+
+✅ Command successful.
+```
+
+At this point, the `mint_authority` is transferred nack to the Candy Machine `authority`. The Candy Guard account remains on-chain and can be reused.
+
+</p>
+</details>
+
+### `withdraw`
+
+To close the Candy Guard account and retrieve the rent fee, use the command:
+
+```
+sugar guard withdraw
+```
+
+<details>
+<summary>Output</summary>
+<p>
+
+```
+[1/2] 🔍 Loading candy guard
+▪▪▪▪▪ Done
+
+[2/2] 🏧 Retrieving funds
+Signature: Yfm2kBbs5kygRzH6QDDTXyx2wxqGZ6P6RkTAHEMkHJ7PvpCZKUu2akkZBDHYfdy1TNyUPLywuNSJoPGWAmTSRuK
+
+Received ◎ 0.00268656 from rent fee.
+
+✅ Command successful.
+```
+
+At this point, the `mint_authority` is transferred nack to the Candy Machine `authority`. The Candy Guard account remains on-chain and can be reused.
```

Comment:
nack -> back

Reply:
Done – actually line 561 should not be there, so I deleted it.